### ISSUE TITLE
How to activate tab in vue2-ace-editor

### ISSUE THREAD

**Issue by klouzoun** [chairuosen/vue2-ace-editor#31]
Hello,

I want to know how to enable tab in you package `vue2-ace-editor`. Currently tab doesn't work with the init config.

Best regards,

Kevin.

**Comment by chairuosen** [comment]
@klouzoun do you mean the keyboard button `tab`, It seems working on my side. Try this demo? http://editor.ruosen.io/
Is tab working?

**Comment by klouzoun** [comment]
Yes tab working in your examples, can you paste requires that you have used ?

**Comment by klouzoun** [comment]
Ok get the same requires. I will search why I can't use tabs and I will put the answer here when I will find. Thanks a lot. @chairuosen

**Comment by chairuosen** [comment]
try includes emmet for html.

**Comment by EmiyaYang** [comment]
@chairuosen `setOption` might be called before `emit` so that we can decide whether to enable `emmet` or not in `init` handler.
```
//^0.0.11 line 78~79
...
 this.$emit("init", editor);
 editor.setOption("enableEmmet", true);
...
```

**Comment by chairuosen** [comment]
@EmiyaYang yes. enableEmmet shouldn't be always true. I'll remove this line.
